aws/cloudfront_website/main.tf

```diff
@@ -23,6 +23,11 @@ resource "aws_cloudfront_distribution" "s3_distribution" {
       https_port             = "443"
       origin_ssl_protocols   = ["TLSv1"]
     }
+
+    custom_error_response {
+      error_code            = 404
+      response_page_path    = "/"
```

Comment:
The path should be a path to a file, we will be returning `/index.html` in almost every case. And don't forget to add `response_code = 200`, to return 200 OK to the client!
With those changes we are ready to go!